netbox/dcim/filtersets.py

```diff
@@ -1304,6 +1304,9 @@ class InterfaceFilterSet(
         to_field_name='rd',
         label='VRF (RD)',
     )
+    is_occupied = django_filters.BooleanFilter(
```

Comment:
This should go under CabledObjectFilterSet so that the filter is available for all object types to which a cable can be attached (those which inherit from CabledObjectModel).